src/modules/search/components/Search.jsx

```diff
@@ -35,7 +35,7 @@ const SearchHolder = styled.div`
 const SearchBackground = styled.div`
 	background: rgba(25, 25, 25, 0.97);
 	width: 100%;
-	height: 100vh;
+	height: ${_ => Math.max(document.querySelector('#queue').offsetHeight, 200) + 'px'};
```

Comment:
@rtluu So the real problem is that the size of this should really depend on the queue. Instead of jumping through a bunch of hoops to do this the right way (i.e. make the search background a part of the queue component) here's a quick hack to size it.

Reply:
gotcha yea that makes sense, that's a cool hack tho! never seen that before